BSP/Simulator/Hardware/simulate.py

```diff
@@ -87,6 +89,9 @@ def update_lights():
         light.set(f"{lights[i]}: {(lights_state >> i) & 0x01}")
     window.after(LIGHTS_FREQ, update_lights)
 
+def update_MotorDisable():
+    """Sends MOTOR_DISABLE message when checkbox is checked/unchecked"""
+    MotorMessage.sendMotorDisable(chargingBool.get())
```

Comment:
Make sure that you add a call to the `charging_frame.after()` method here so it calls this method periodically. Right now it's only calling this method when the checkmark is pressed. We want the BPS message to be continuously broadcasted on a periodic interval all the time. The only thing that changes is the data packet.